src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1591,11 +1593,9 @@
         },
 
         "graphs": {
```

Comment:
>graphs [](start = 9, length = 6)

untest part of the format alert. :) that's why we hadn't uncovered this. if we have no uptake of graphs, should consider dropping entirely.